pandas/tests/series/test_constructors.py

```diff
@@ -79,17 +80,34 @@ def test_constructor(self):
         m = MultiIndex.from_arrays([[1, 2], [3, 4]])
         pytest.raises(NotImplementedError, Series, m)
 
-    def test_constructor_empty(self):
+    @pytest.mark.parametrize('input_class', [list, dict, OrderedDict])
+    def test_constructor_empty(self, input_class):
         empty = Series()
-        empty2 = Series([])
-
-        # the are Index() and RangeIndex() which don't compare type equal
+        empty2 = Series(input_class())
+        # these are Index() and RangeIndex() which don't compare type equal
         # but are just .equals
         assert_series_equal(empty, empty2, check_index_type=False)
 
-        empty = Series(index=lrange(10))
```

Comment:
so I think these tests got eliminated? can you add another test (or add onto the construction via ``np.nan, None, float('nan')`` which hits this case?

Reply:
This test is wrong (dtype _shouldn't_ be ``float64``), the right one would fail, see #17261 . Adding an xfailing test.

Reply:
The xfailing test would basically be this one with explicit dtype, it's better to change it when fixing #17261 . Adding a fixed test for this instead.

Reply:
done, ping